src/pages/Proxies.tsx

```diff
@@ -192,7 +207,12 @@ export default () => {
                     <div class="space-y-2">
                       <div class="flex items-center justify-between pr-8">
                         <div class="flex items-center">
-                          <Show when={proxyGroup.icon}>
+                          <Show
```

Comment:
If you want to replicate `Ternary Operator` in jsx with `Show` component, you can put `else` case component in `fallback` property. See https://docs.solidjs.com/reference/components/show

Reply:
Done